geoh5py/shared/concatenation/drillhole.py

```diff
@@ -254,7 +254,7 @@ def validate_depth_data(
         else:
             out_group = property_group
 
-        _ = getattr(self, "add_data")(
+        _ = self.add_data(
```

Comment:
Why having _ = before?
Just add data is enought

Reply:
Nice catch. Removing.